flink-table-store-hive/flink-table-store-hive-catalog/src/test/java/org/apache/flink/table/store/hive/HiveCatalogITCase.java

```diff
@@ -317,6 +319,29 @@ public void testHiveLock() throws InterruptedException {
         assertThat(count.get()).isEqualTo(100);
     }
 
+    @Test
+    public void testHiveTableLowerCase() throws Exception {
+        // create table
+        tEnv.executeSql("CREATE TABLE T ( A INT, b STRING ) WITH ( 'file.format' = 'avro' )")
+                .await();
+        tEnv.executeSql("CREATE TABLE S ( a INT, B STRING ) WITH ( 'file.format' = 'avro' )")
+                .await();
+
```

Comment:
`tEnv.executeSql("SELECT A, B FROM t").print();` will fail. Because Flink is not case insensitive.